### ISSUE TITLE
It's not work when datatype is enum

### ISSUE THREAD

**Issue by 12789** [sdispater/orator#40]
![image](https://cloud.githubusercontent.com/assets/244297/13023366/1aa395e2-d19d-11e5-90a0-a20a02f602eb.png)

**Comment by sdispater** [comment]
When does this happen?

**Comment by joelbarker2011** [comment]
This occurs when trying to create a record with an enum attribute. Test case:

```python
# test.py
from os import environ as env

from orator import DatabaseManager
from orator import Model
from orator import Schema

class Widget(Model):
    pass

if __name__ == '__main__':
    config = {
        'test': {
            'driver': 'mysql',
            'user': env['DB_USER'],
            'password': env['DB_PASS'],
            'host': env.get('DB_HOST', 'localhost'),
            'database': env['DB_NAME']
        }
    }

    db = DatabaseManager(config)
    Model.set_connection_resolver(db)
    schema = Schema(db)

    schema.drop_if_exists('widgets')

    with schema.create('widgets') as t:
        t.increments('id')
        t.enum('material', ['ANIMAL', 'VEGETABLE', 'MINERAL'])

    wally = Widget.create(id=42, material='ANIMAL')

# run like this:
# DB_USER=mydbuser DB_PASS=mydbpass DB_NAME=mydatabase python test.py
```

I can work around it by monkey-patching orator:
```python
from orator.dbal.platforms import MySQLPlatform
MySQLPlatform.INTERNAL_TYPE_MAPPING['enum'] = 'enum'
```

The proper solution would be adding a line to _orator/dbal/platforms/mysql_platform.py_:
```python
    INTERNAL_TYPE_MAPPING = {
        'tinyint': 'boolean',
# ...
        'set': 'simple_array',
        'enum': 'enum',    # <- new line
    }
```
The same line probably should be added to _postgres_platform.py_ and/or _sqlite_platform.py_ (I haven't tested with Postgres or Sqlite).

**Comment by sdispater** [comment]
Fixed in the latest 0.8.1 version

**Comment by joelbarker2011** [comment]
Confirmed that this has been fixed (commit 6c3d4a17189686f7bfc86f27904800fb84647ce2). Thanks!
